staging/src/k8s.io/client-go/datapartition/apiserverconfigmanager.go

```diff
@@ -339,9 +338,12 @@ func StartAPIServerConfigManager(endpointsInformer coreinformers.EndpointsInform
 }
 
 func StartAPIServerConfigManagerAndInformerFactory(client clientset.Interface, stopCh <-chan struct{}) {
+	fieldSelector := fields.Set{"metadata.name": types.KubernetesServiceName,
```

Comment:
Better to have a naming convention in the system to uniquely naming the Api Server end points , so there won't need those ns and tenant filters.